source/rtos2/Include/RTX_Config.h

```diff
@@ -57,7 +57,7 @@
 //   <e>Round-Robin Thread switching
 //   <i> Enables Round-Robin Thread switching.
 #ifndef OS_ROBIN_ENABLE
-#define OS_ROBIN_ENABLE             0 // DAPLINK. Default was: 1
+#define OS_ROBIN_ENABLE             1 // DAPLINK. Default was: 1
```

Comment:
We don't want to enable this for all other HICs. Can we just set the macro in the stm32h743ii_if yaml?